models/gc_grt123_lung_cancer/meta.json

```diff
@@ -71,19 +71,35 @@
     "use": {
       "title": "Intended use",
       "text": "This algorithm analyzes non-contrast CT scans of the thorax and predicts the lung cancer risk. ",
-      "references": [],
+      "references": [
+        {
+          "label": "Lung cancer risk estimation algorithm on grand-challenge",
+          "url": "https://grand-challenge.org/algorithms/dsb2017-grt123/"
+        }
+      ],
       "tables": []
     },
     "analyses": {
-      "title": "",
-      "text": "",
+      "title": "Evaluation",
+      "text": "The nodule detection was evaluated on the validation set of DSB. It contains data from 198 cases and there are 71 (7 nodules smaller than 6 mm are ruled out) nodules in total. The Free Response Operating Characteristic (FROC) is used to evaluate the performance and can be viewed in the publication. The Case classification is evaluated using the Area Under the Curve (AUC) metric.",
       "references": [],
-      "tables": []
+      "tables": [{
+        "label": "AUC for the Case classification",
+        "entries": {
+          "AUC training set": 0.90,
+          "AUC test set": 0.87
+        }
+      }]
     },
     "evaluation": {
-      "title": "",
-      "text": "",
-      "references": [],
+      "title": "Evaluation data",
+      "text": "The model was evaluated against a private dataset of 300 low-dose CT images. 150 patient scans were from the competition set and 150 were from an independent dataset. Both test datasets contained 50 cancer-positive scans and 100 cancer-negative scans.",
```

Comment:
If either the competition set or the independent one is publicly available, could we add a reference?
Otherwise, I'm wondering if the distinction is of relevance.

Reply:
No, these are private, I'll remove the distinction.